### DISCUSSION TITLE
User Management Starter

### DISCUSSION THREAD

**Discussion by OsmanM94** [supabase/supabase#27643] [Questions] [answered]
Can someone please help me as i'm running out of fuel to understand this. I'm new to Supabase. I'm using updating the User Management Starter to fit my own requirements as per photos but Signing in with Apple gives me an error in console "Internal Server Error".  Using the default User Management Starter works like charm, i can sign in and out no issues. Surely i do something wrong in the SQL Editor, please help me understand.

/// THIS IS WHAT I'm TRYING TO ACHIEVE
![My userManagement](https://github.com/supabase/supabase/assets/168109739/fc53690b-aab1-4dee-898b-add81bc37195)

/// THIS IS THE DEFAULT USER MANAGEMENT FROM SUPABASE THAT WORKS
![Default userManagement](https://github.com/supabase/supabase/assets/168109739/b5b70f59-ea5a-449d-b156-e593f2cda208)

/// THIS IS THE TABLE I WANT TO CREATE
![my table](https://github.com/supabase/supabase/assets/168109739/28d09dc2-375c-4e88-ad11-e65f06a47710)

What i'm doing wrong in the SQL Editor? thank you

**Comment by GaryAustin1** [comment]
The best thing to do is to look in the Postgres logs of the dashboard and see what your error is.

**Reply by GaryAustin1** [reply, marked as answer]
One obvious thing is you are using new.id (the user id) for your bigint id column which won't work.  You really don't need a primary bigint and should use the user_id column as the primary key for the profiles table.   But if you want that then don't set the id at all in your insert and let the sequence fill it in automatically.

also your username column is unique, but you are sending a fixed string which will fail the 2nd time.

**Reply by OsmanM94** [reply]
bless you man. I didn't know about that. I get this:  column "id" is of type bigint but expression is of type uuid

**Reply by GaryAustin1** [reply]
Beat you to it.

**Comment by OsmanM94** [comment]
You are a legend bud it works brilliant, ain't gonna lie i was struggling for the past 2 days to get this working. Originally i have upgraded my plan to PRO i was suspecting i'm API limited somehow hahah... thank you again
